redis/src/main/scala/zio/redis/codec/CRC16.scala

```diff
@@ -0,0 +1,43 @@
+package zio.redis.codec
+
+import zio.Chunk
+
+/**
```

Comment:
Super nitpicky:
```suggestion
/*
```

Reply:
But, we everywhere in documentation use 2 stars. It's necessary to be green and formaIted like documentation in IntelIJ Idea)

Reply:
For public scaladoc, yes. However, this thing is (package) private and I'd argue that it should not be present there.

Reply:
In other redis libraries it's present, but ok i would rid of one star)